libs/ui/src/lib/step-bar/step-bar.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing'
+
+import { StepBarComponent } from './step-bar.component'
+
+describe('StepBarComponent', () => {
+  let component: StepBarComponent
+  let fixture: ComponentFixture<StepBarComponent>
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [StepBarComponent],
+    }).compileComponents()
+  }))
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(StepBarComponent)
+    component = fixture.componentInstance
+    fixture.detectChanges()
+  })
+
+  it('should create', () => {
```

Comment:
Tests please.
Check that 

- we have the correct number of steps
- That the current steps color is correct
- That other step colors are correct